sdk/keyvault/keyvault-certificates/recordings/node/certificates_client__create_read_update_and_delete/recording_can_get_a_certificates_secret_in_pkcs_12_format.js

```diff
@@ -1,3036 +1,5 @@
 let nock = require('nock');
 
-module.exports.hash = "74ba7776254a2ca9a10c63f3155804bb";
+module.exports.hash = "4c366048af14c704af26e981272b9441";
```

Comment:
We can probably just delete this file

Reply:
The recorder seems to require it

Reply:
because the test doesn't know it is skipped until the skip call is seen.

Reply:
Yes I recall that now - thanks!